### ISSUE TITLE
NotificationsIOS.checkPermissions() is not a function

### ISSUE THREAD

**Issue by gino8080** [wix/react-native-notifications#105]
Using the example code 

```javascript
NotificationsIOS.checkPermissions().then((currentPermissions) => {
    console.log('Badges enabled: ' + !!currentPermissions.badge);
    console.log('Sounds enabled: ' + !!currentPermissions.sound);
    console.log('Alerts enabled: ' + !!currentPermissions.alert);
});
```

i got this error
```javascript
ExceptionsManager.js:73 Unhandled JS Exception: _reactNativeNotifications2.default.checkPermissions is not a function
```

this is my full index.ios.js

```javascript
/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 * @flow
 */

import React, { Component } from "react";
import {
  AppRegistry,
  StyleSheet,
  Text,
  View,
  TouchableHighlight
} from "react-native";
import NotificationsIOS from "react-native-notifications";

export default class pushnotifications extends Component {
  constructor() {
    super();

    console.info("CONSTRUCTOR");
    NotificationsIOS.addEventListener(
      "remoteNotificationsRegistered",
      this.onPushRegistered.bind(this)
    );
    NotificationsIOS.addEventListener(
      "remoteNotificationsRegistrationFailed",
      this.onPushRegistrationFailed.bind(this)
    );
    NotificationsIOS.requestPermissions();
  }

  onPushRegistrationFailed(error) {
    // For example:
    //
    // error={
    //   domain: 'NSCocoaErroDomain',
    //   code: 3010,
    //   localizedDescription: 'remote notifications are not supported in the simulator'
    // }
    console.info("ERRORE", error);
  }

  componentWillUnmount() {
    // prevent memory leaks!
    NotificationsIOS.removeEventListener(
      "remoteNotificationsRegistered",
      this.onPushRegistered.bind(this)
    );
    NotificationsIOS.removeEventListener(
      "remoteNotificationsRegistrationFailed",
      this.onPushRegistrationFailed.bind(this)
    );
  }

  checkNotif() {
    NotificationsIOS.checkPermissions().then(currentPermissions => {
      console.log("Badges enabled: " + !!currentPermissions.badge);
      console.log("Sounds enabled: " + !!currentPermissions.sound);
      console.log("Alerts enabled: " + !!currentPermissions.alert);
    });
  }
  onPushRegistered(deviceToken) {
    console.log("Device Token Received", deviceToken);
  }

  render() {
    return (
      <View style={styles.container}>
        <Text style={styles.welcome}>+++ Welcome to React Native!</Text>
        <TouchableHighlight onPress={() => this.checkNotif()}>
          <Text style={styles.instructions}>CHECK NOTIF</Text>
        </TouchableHighlight>

        <Text style={styles.instructions}>
          Press Cmd+R to reload,{"\n"}
          Cmd+D or shake for dev menu
        </Text>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: "center",
    alignItems: "center",
    backgroundColor: "#F5FCFF"
  },
  welcome: {
    fontSize: 20,
    textAlign: "center",
    margin: 10
  },
  instructions: {
    textAlign: "center",
    color: "#333333",
    marginBottom: 5
  }
});

AppRegistry.registerComponent("pushnotifications", () => pushnotifications);
```

**Comment by adrienthiery** [comment]
Did you `react-native link react-native-notifications`, added the code to `AppDelegate.m` and re-run `react-native run-ios` ?

This happens usually when `NativeRNNotifications` is null [here](https://github.com/wix/react-native-notifications/blob/e4921f36b77edeb0b4ee7d119c628fd4235be954/index.ios.js#L217) and that's when the native part of the code has not been installed properly.

**Comment by reberhardt7** [comment]
Ahh sorry, this code is from a recent commit that hasn't made it to npm yet. (cc @gran33) For now, you can update your package.json to use the latest commit:

```
    "react-native-notifications": "wix/react-native-notifications#e4921f36",
```

**Comment by gran33** [comment]
Sorry for the delay, published and available in `v1.1.14`.

Enjoy guys! 👨‍💻

**Comment by gino8080** [comment]
i can confirm now its working!
